### PR TITLE
New left nav for devsite

### PR DESCRIPTION
**Pull request by balopat**
Redefines the left nav for the new site according to our discussions.

(Googlers please refer to go/qpolish-release-tracker "Cirq docs" tab for the left nav structure, I also staged the version in go/q-stage)

### CODE REVIEW

**Comment by balopat** [comment]
cc @lamberta PTAL

File: docs/_book.yaml [hunk 1]
```diff
@@ -27,92 +27,112 @@ upper_tabs:
   lower_tabs:
     # Subsite tabs
     other:
-    - name: "Tutorials"
+    - name: "Guides"
```
Comment (lamberta) [inline docs/_book.yaml:30]: I support moving the guide up, but think it should be "Guide" (singular) to keep it consistent with the other projects ("Guide & Tutorials"). There is one Cirq developer guide (that contains multiple sections and pages)

Comment (balopat) [reply docs/_book.yaml]: Sounds good!

File: docs/_book.yaml [hunk 2]
```diff
@@ -27,92 +27,112 @@ upper_tabs:
...
-        path: /cirq/tutorials/google/reservations
+
       ####  PASQAL ####
-      - heading: "Pasqal"
-      - title: "Quantum Circuits on Pasqal Devices"
-        path: /cirq/pasqal/getting_started
+      - heading: "Pasqal hardware"
       - title: "Pasqal devices"
         path: /cirq/pasqal/devices
       - title: "Pasqal sampler"
```
Comment (lamberta) [inline docs/_book.yaml:93]: Perhaps for another change, but I was initially confused what a "Pasqal sampler" was---is this a funny name for a demo/tutorial? (Looking at the doc I see this is not the case :)

**Review by lamberta — APPROVED** [review]

File: docs/_book.yaml (hunk 2 above)
Comment (balopat) [reply docs/_book.yaml:93]: This is more of a reference as far as I'm concerned. Describes one important concept of the Pasqal API. Why do you think this is a demo/tutorial? It's under Guide.

Comment (lamberta) [reply docs/_book.yaml:93]: It was more the "sampler" name that threw me a bit. Not a big deal just caught my attention.

Comment (balopat) [reply docs/_book.yaml:93]: Ah, I see, the technical term collides with the regular term "sampler" :D I am so used to it it didn't even occur to me. I'll think about alternate wording, thank you!

**Review by dstrain115 — APPROVED** [review]

File: docs/_book.yaml [hunk 3]
```diff
@@ -27,92 +27,112 @@ upper_tabs:
...
+      - title: "Textbook examples"
+        path: /cirq/tutorials/examples
+      - title: "Quantum variational algorithm"
+        path: /cirq/tutorials/variational_algorithm
+      - title: "QAOA experiment"
+        path: /cirq/tutorials/QAOA_Demo
+      - title: "Hidden linear function problem"
+        path: /cirq/tutorials/hidden_linear_function
+      - title: "Quantum walks"
+        path: /cirq/tutorials/Quantum_Walk
```
[the remark this answers is not in the document]
Comment (balopat) [reply docs/_book.yaml:111]: Yeah, it bothered me too, I guess this is a great time to fix it. Thanks for pointing it out!

File: docs/gates.ipynb [hunk 4]
```diff
@@ -37,7 +37,7 @@
     "id": "xwec7FrkdFmi"
    },
    "source": [
-    "# Gates"
+    "# Gates and Operations"
```
[the remark this answers is not in the document]
Comment (balopat) [reply docs/gates.ipynb:40]: Oh right, I did that fix in all the navs at least :D Good catch!

**Pull request merged**